### ISSUE TITLE
Italian carriers

### ISSUE THREAD

**Issue by DarkSkull91** [0x192/Universal-Android-Debloater#79]
It would be nice to add also italian carriers apps

**Comment by 0x192** [comment]
I agree but you are probably in a better position than me to do so. I am not Italian and know nothing about Italian ISPs.

Feel free to list and document those apps here so that I can add it (or better, do directly a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)!)

**Comment by ghost** [comment]
Here's a partial list of carrier apps (please note that I'm not using any of these, comments/review would be appreciated):

- it.telecomitalia.centodiciannove
- it.telecomitalia.timpersonal
- it.telecomitalia.iotim
- it.telecomitalia.cubovision
- it.telecomitalia.comunicazioneintegratabroadsoft
- it.vodafone.my190
- it.wind.myWind
- com.locationlabs.familyshield.wind
- it.iliad.android.iccid
- it.iliad.android.vvm
- it.homobile.ho
- it.very.mobile
- it.kenamobile.kenamobile
- com.tiscali.android.my130
- com.lycadigital.lycamobile
- com.rabona.myshoo
- com.rabonamobile.rabona
- com.massresponse.spusuit
- it.fastweb.myfastweb
